### PR TITLE
updated twitter.py and ronaAnalysis-ncdc.py

### LINKED ISSUE #10: Update ncdc data scrapping

### PR DESCRIPTION
**Pull request by SLKhadeeja**
fixes #10

### CODE REVIEW

File: Scripts/Activate.ps1 [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        Write-Host -NoNewline -ForegroundColor Green '(pandelytics) '
+        _OLD_VIRTUAL_PROMPT
+    }
+}
+
+# Clear PYTHONHOME
+if (Test-Path env:PYTHONHOME) {
+    copy-item env:PYTHONHOME env:_OLD_VIRTUAL_PYTHONHOME
+    remove-item env:PYTHONHOME
+}
```
Comment (deven96) [inline Scripts/Activate.ps1]: You can exclude your entire scripts folder and all virtualenv related stuff by removing it from your git cache and adding it to your gitignore

File: pyvenv.cfg [hunk 2]
```diff
@@ -0,0 +1,3 @@
+home = C:\Users\ladankhadija\AppData\Local\Programs\Python\Python37
+include-system-site-packages = false
+version = 3.7.6
```
Comment (deven96) [inline pyvenv.cfg]: Also goes in gitignore

**Review by deven96 — COMMENTED** [review]
Is the ncdc parser working now?

**Comment by SLKhadeeja** [comment]
It's working now

Sorry.... I forgot about the virtual env.... I'll fix it

File: ronaAnalysis-ncdc.py [hunk 3]
```diff
@@ -1,32 +1,35 @@
...
-    allStateCases = allStateCases.set_index('States')
-    allStateCases = allStateCases.drop(np.nan)
-
-    allStateCases_json = allStateCases.to_json(orient = 'index')
-    return (allStateCases_json)
-
-if __name__ == '__main__':
-	app.run(debug=True)
\ No newline at end of file
+import requests
```
Comment (MeNsaaH) [inline ronaAnalysis-ncdc.py]: Could you please move this implementation to the ncdc file https://github.com/Pandelytics/pandelytics/blob/master/pandelytics/engines/ncdc.py

Use the class base approach

**Review by MeNsaaH — CHANGES_REQUESTED** [review]

**Review by MeNsaaH — DISMISSED** [review]

**Comment by MeNsaaH** [comment]
@SLKhadeeja could you resolve the merge conflicts. On you branch execute
```bash
	git fetch origin master
	git merge origin/master
```
Resolve the conflicts that occur and push them

**Review by MeNsaaH — APPROVED** [review]

**Pull request merged**